libraries/chain/controller.cpp

```diff
@@ -1367,18 +1392,12 @@ struct controller_impl {
       } catch( const objective_block_validation_exception& ) {
          throw;
       } catch( const fc::exception& e ) {
-         cpu_time_to_bill_us = trx_context.update_billed_cpu_time( fc::time_point::now() );
-         trace->error_code = controller::convert_exception_to_error_code( e );
-         trace->except = e;
-         trace->except_ptr = std::current_exception();
-         trace->elapsed = fc::time_point::now() - trx_context.start;
-
-         if (auto dm_logger = get_deep_mind_logger()) {
-            fc_dlog(*dm_logger, "DTRX_OP FAILED ${action_id}",
-               ("action_id", trx_context.get_action_id())
-            );
-         }
+        handle_exception(e);
+      } catch ( const std::exception& e) {
```

Comment:
Here and elsewhere. This now catches `boost::interprocess::bad_alloc` which we want to handle differently.

Reply:
Good catch.  I was too focused on the std::exception/fc::exception changes.

Reply:
So basically in these cases where I've added `std::exception`, handle the boost bad_alloc and std bad_alloc as well to just bubble up the exception?